chaosaws/ec2/actions.py

```diff
@@ -12,13 +12,13 @@
 from logzero import logger
 
 __all__ = ["stop_instance", "stop_instances", "terminate_instances",
-           "terminate_instance"]
+           "terminate_instance", "start_instances", "reboot_instances"]
 
 
 def stop_instance(instance_id: str = None, az: str = None, force: bool = False,
                   filters: List[Dict[str, Any]] = None,
                   configuration: Configuration = None,
-                  secrets: Secrets = None) -> AWSResponse:
+                  secrets: Secrets = None) -> List[AWSResponse]:
```

Comment:
Is that expected on a single `stop_instance`?

Reply:
Yes, the response is always a list. The private function called to stop it appends results to a list to return.

Reply:
Gotcha. Thanks.